apps/dashboard/src/@3rdweb-sdk/react/hooks/useEngine.ts

```diff
@@ -401,29 +415,22 @@ export function useEngineTransactions(instance: string, autoUpdate: boolean) {
   });
 }
 
-type WalletConfig =
-  | {
-      type: "local";
-    }
-  | {
-      type: "aws-kms";
-      awsAccessKeyId: string;
-      awsSecretAccessKey: string;
-      awsRegion: string;
-    }
-  | {
-      type: "gcp-kms";
-      gcpApplicationProjectId: string;
-      gcpKmsLocationId: string;
-      gcpKmsKeyRingId: string;
-      gcpApplicationCredentialEmail: string;
-      gcpApplicationCredentialPrivateKey: string;
-    };
+export interface WalletConfigResponse {
```

Comment:
Can we bring back union type here so it's clear which combinations are possible/not possible

Reply:
Hmm the endpoint now doesn't accept a union type which is why I matched it on the dashboard side.